src/main/scala/org/broadinstitute/dsde/agora/server/webservice/handlers/QueryHandler.scala

```diff
@@ -19,60 +19,58 @@ import scala.concurrent.{ExecutionContext, Future}
  * It then handles the returns from the business layer and completes the request. It is responsible for querying the
  * methods repository for methods and method configurations.
  */
-class QueryHandler(dataSource: PermissionsDataSource, implicit val ec: ExecutionContext) extends Actor {
-  implicit val system = context.system
-
+class QueryHandler(dataSource: PermissionsDataSource)(implicit ec: ExecutionContext) {
```

Comment:
I think this class should go away. The usage of it is a direct passthrough to`agoraBusiness.find`

Reply:
Agreed. Will take care of it as part of the next PR.